gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/BuildMasterNotFound.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 Netflix, Inc.
```

Comment:
This should be you or your company.  (And the same for copyright headers in other files.)

Reply:
@ezimanyi Done. =)